### PR TITLE
documentation fix

### PR DESCRIPTION
**Pull request by gAmUssA**
- added testing prerequisites
- ignore java artifacts in repository

@hazelcast-incubator/hazelcast-nodejs-client-team please, review and merge

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -16,6 +16,23 @@ npm install
...
++
+
+----
+curl -OL https://jcenter.bintray.com/com/hazelcast/hazelcast-all/3.6-EA2/hazelcast-all-3.6-EA2.jar
+----
+* Running Hazelcast members
+
++
+----
+java -showversion -cp "$PWD/hazelcast-all-3.6-EA2.jar:" com.hazelcast.core.server.StartServer
```
Comment (GhostInAMachine) [inline README.adoc]: Remove this line, we will have to launch the instances with different configurations

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
 node_modules/*
-*.iml
\ No newline at end of file
+*.iml
+
+### Java ###
+*.class
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
```
Comment (GhostInAMachine) [inline .gitignore]: We certainly will have a lot of files from J2ME :D

File: README.adoc (hunk 1 above)
Comment (gAmUssA) [inline README.adoc]: this commands work for default test run.

- run curl (if jar is not there)
- run java
- run `npm test`
- PROFIT!

**Comment by gAmUssA** [comment]
updated PR with upstream changes.
addressed @GhostInAMachine comment of ignoring javaME artifacts.

**Pull request merged**